irohad/torii/query_service.hpp

```diff
@@ -60,9 +60,10 @@ namespace iroha {
       void Find(iroha::protocol::Query const &request,
                 iroha::protocol::QueryResponse &response);
 
-      grpc::Status Find(grpc::ServerContext *context,
-                        const iroha::protocol::Query *request,
-                        iroha::protocol::QueryResponse *response) override;
+      virtual grpc::Status Find(
```

Comment:
`virtual` should not be required because the method is already marked as `override`.